dohproxy/utils.py

```diff
@@ -41,6 +42,32 @@ def extract_path_params(url: str) -> Tuple[str, Dict[str, List[str]]]:
     return p.path, params
 
 
+def create_custom_ssl_context(
+        insecure: bool,
```

Comment:
and here use the following signature:
```
def create_custom_ssl_context(
    *,
    insecure: bool,
    cafile: Optional[str] = None) -> ssl.SSLContext:
```

this should force the use of keyword arguments for both insecure and cafile